aries-site/src/pages/templates/toast-notifications.mdx

```diff
@@ -15,7 +19,13 @@ import { Box } from 'grommet';
 
 ## Guidance
 
-Everything that you need to know about toasts and more.
+Toasts are a low-attention notification mechanism primarily for use in confirming task receipt and completion.
```

Comment:
I'm not sure what "low-attention" means. Does it mean that it does not call attention to itself very much?